index.js

```diff
@@ -0,0 +1,67 @@
+const {audioProcess} = require('./lib/transcription');
+const axios = require('axios');
+const {redactSensitiveInfo} = require('./lib/audioRedaction');
+const path = require('path');
+const fs = require('fs-extra');
+require('dotenv').config();
+
+const audioDir = process.env.AUDIO_DIR;
+const accountSid = process.env.ACCOUNT_SID;
+const callSid = process.env.CALL_SID;
+const jambonzApiToken = process.env.JAMBONZ_API_TOKEN;
+const recordingDate = '2024/06/06';
+
+const audioURL = `https://jambonz.one/api/v1/Accounts/${accountSid}/RecentCalls/${callSid}/record/${recordingDate}/mp3`;
+console.log(audioURL);
+
+const config = {
+  responseType: 'stream',
+  headers: {
+    Authorization: `Bearer ${jambonzApiToken}`
+  }
+};
+
+// Function to download and process audio
+async function downloadAndProcessAudio() {
+  try {
+    const response = await axios.get(audioURL, config);
+    const audioFilePath = path.join(audioDir, 'recording.mp3');
+    const writer = fs.createWriteStream(audioFilePath);
+
+    response.data.pipe(writer);
```

Comment:
When we are using streams in Node.js -- which is to say we want to process a stream of incoming data and possibly process it via a series of functions in sequence, we pretty much always want to use `pipeline` ([as shown here](https://github.com/jambonz/jambonz-api-server/blob/6041b1d595225c28871665ee014a85017fdb1330/lib/record/upload.js#L66)), not `pipe`.

Streams in Node.js are complex, often misused, but they are the only way to scale when processing potentially large files.  Your tests so far are on small files but this library will need to include a test showing that it can process several hour-long conversations at the same time in order to prove it is ready for use.

Please watch [this video in its entirety](https://www.youtube.com/watch?v=edB964-YYpE&list=PL_x4nRdxj60K2j9l_SflmqmaTA2XVBuqc) by Matteo Collina who knows the inner workings of streams better than anyone and does a great job of pointing out its proper usage